internal/provider/resource_application.go

```diff
@@ -48,6 +49,17 @@ const (
 	StorageKey          = "storage"
 	UnitsKey            = "units"
 
+	imageRegistriesMarkdownDescription = `
+	Image registry credentials for OCI images specified in the charm resources. The map key is the registry URL.
```

Comment:
We're sometimes saying OCI image and sometimes just image, can we keep it consistent as "OCI Image" (and "OCI Registry"). I also wouldn't ever just name it "Image" because that could be any kind of image, vm, png 😉, etc..

Reply:
Will use OCI image everwhere.